Cargo.toml

```diff
@@ -29,4 +34,4 @@ needless_return = "allow"
 
 [[bin]]
 name = "litra"
-required-features = ["cli"]
```

Comment:
Let's make CLI a required feature, and then conditionally include the MCP stuff if the MCP feature is enabled (which it should be by default)